docs/FeatureManagement/FeatureFlagv2.0.0.schema.json

```diff
@@ -0,0 +1,354 @@
+                }
+            },
+            "percentile": {
+                "$id": "#/properties/allocation/properties/percentile",
+                "type": "array",
+                "title": "Percentile Allocation Collection",
+                "description": "A list of objects each containing a variant name and percentage range for which that variant should be used.",
+                "items": {
+                    "$id": "#/properties/allocation/properties/percentile/items",
+                    "type": "object",
+                    "title": "Percentile Allocation",
+                    "required": [
+                        "variant",
+                        "from",
+                        "to"
+                    ],
+                    "properties": {
+                        "variant": {
+                            "$id": "#/properties/allocation/properties/percentile/items/properties/variant",
+                            "type": "string",
+                            "title": "Percentile Allocation Variant",
+                            "description": "The name of the variant to use if the calculated percentage for the current user falls in the provided range.",
+                            "default": "",
+                            "pattern": "^(.*)$"
+                        },
+                        "from": {
+                            "$id": "#/properties/allocation/properties/percentile/items/properties/from",
+                            "type": "number",
+                            "title": "Percentile Allocation From",
+                            "description": "The lower end of the percentage range for which this variant will be used.",
+                            "default": 0
+                        },
+                        "to": {
+                            "$id": "#/properties/allocation/properties/percentile/items/properties/to",
+                            "type": "number",
+                            "title": "Percentile Allocation To",
+                            "description": "The upper end of the percentage range for which this variant will be used.",
+                            "default": 100
+                        }
+                    }
+                }
+            },
+            "seed": {
+                "$id": "#/properties/allocation/properties/seed",
+                "type": "string",
+                "title": "Percentile Allocation Seed",
+                "description": "The value which percentile calculations are based on. Calculated percentages are consistent across features if the user and seed are consistent.",
+                "default": "",
+                "pattern": "^(.*)$"
+            }
+        }
+      },    
+      "telemetry_enabled": {
+        "$id": "#/properties/telemetry_enabled",
+        "type": "boolean",
+        "title": "Feature Telemetry Enabled State",
+        "description": "A flag to enable or disable sending telemetry events.",
```

Comment:
Should we drop the word "events"? I think "sending telemetry" is enough.

Reply:
Sure, bit more concise